src/BaseWrapper.sol

```diff
@@ -46,9 +46,6 @@ abstract contract BaseWrapper is IERC7399 {
         return result;
     }
 
-    /// @dev Call the flashloan function in the child contract
-    function _flashLoan(address asset, uint256 amount, bytes memory params) internal virtual;
-
     /// @dev Handle the common parts of bridging the callback
     function bridgeToCallback(address asset, uint256 amount, uint256 fee, bytes memory params) internal {
```

Comment:
should we take the chance to add `_` to the name of this internal function?

Reply:
Fixed

Reply:
Fixed